nucleus/common/common-util/src/main/java/com/sun/common/util/logging/LoggingConfigImpl.java

```diff
@@ -385,7 +385,7 @@ public synchronized Map<String, String> deleteLoggingProperties(final Collection
         LOG.log(Level.INFO, "deleteLoggingProperties(properties={0})", properties);
         loadLoggingProperties();
         remove(properties);
-        rewritePropertiesFileAndNotifyMonitoring();
+        rewritePropertiesFileAndNotifyMonitoring("");
```

Comment:
`null` would be better

Reply:
Thank you for your suggestion. 
But existing code uses  `""` instead of `null` as below, and using `""` and `null` mixedly would not be desirable...

https://github.com/eclipse-ee4j/glassfish/blob/858cfe603f66c1e2c5b4d9cbf10baa9da01ac3e7/nucleus/common/common-util/src/main/java/com/sun/common/util/logging/LoggingConfigImpl.java#L235

https://github.com/eclipse-ee4j/glassfish/blob/858cfe603f66c1e2c5b4d9cbf10baa9da01ac3e7/nucleus/common/common-util/src/main/java/com/sun/common/util/logging/LoggingConfigImpl.java#L301

If we want to refactor this code, method overloading (i.e. defining `rewritePropertiesFileAndNotifyMonitoring()` and `closePropFile()` without arguments) would be better rather than using `""` and `null`.
Since doing such refactoring in the current pull request obscures the essense of the bug fix, I think we can do it in another pull request if needed.

Reply:
I would just replace all those empty strings with `null` (and in the future this class should never get an empty string target, it should be resolved when parsing the user input). Adding overloaded methods doesn't bring a value in private methods. Refactoring is ok if you put it to separate commit from the fix.

Additional PR means more work to each of us and waiting on CI ;-)

Reply:
Okay, I've modified the code to use `null` instead of `""`.